extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -614,5 +628,3 @@ Type attribute value: `webtrekk_v2`
 <span style="text-decoration: line-through;">`webtrekk`</span> is deprecated. **Use `webtrekk_v2` instead.**
 
 [/tip]
-
-Adds support for Webtrekk. Configuration details can be found at [supportcenter.webtrekk.com](https://supportcenter.webtrekk.com/en/public/amp-analytics.html).
```

Comment:
Why are we removing this section?

Reply:
This URL no longer exists, the company was acquired.

Reply:
Should we remove the rest of the section rather than just the blurb?